server/sonar-web/src/main/js/api/quality-profiles.ts

```diff
@@ -129,3 +129,43 @@ export function associateProject(profileKey: string, projectKey: string): Promis
 export function dissociateProject(profileKey: string, projectKey: string): Promise<void> {
   return post('/api/qualityprofiles/remove_project', { profileKey, projectKey });
 }
+
+export interface SearchUsersParameters {
+  language: string;
+  organization?: string;
+  profile: string;
+  q?: string;
+  selected: boolean;
+}
+
+export type SearchUsersResponse = Array<{
+  avatar?: string;
+  login: string;
+  name: string;
+}>;
+
+export function searchUsers(_parameters: SearchUsersParameters): Promise<SearchUsersResponse> {
+  // return getJSON('/api/qualityprofiles/search_users', parameters);
```

Comment:
I saw your comment on the PR concerning mock data, but there are the stubs from Julien no ?

Reply:
PS : don't forget the `throwGlobalError` when switching to the real WS